src/OmniSharp.Host/Internal/CommandOptionExtensions.cs

```diff
@@ -1,11 +1,11 @@
 using System.ComponentModel;
 using Microsoft.Extensions.CommandLineUtils;
 
-namespace OmniSharp.Host.Internal
+namespace OmniSharp.Internal
 {
-    internal static class CommandOptionExtensions
+    public static class CommandOptionExtensions
     {
-        internal static T GetValueOrDefault<T>(this CommandOption opt, T defaultValue)
+        public static T GetValueOrDefault<T>(this CommandOption opt, T defaultValue)
```

Comment:
we ended up with public class in an internal namespace 😃

Reply:
doesn't make it right. That's a smell.

What semantic meaning are you trying to convey by having a public class in an internal namespace. Why does this need to be public at all? Is it part our public API surface area?